crypto/bls12381/utils.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package bls12381
+
+import (
+	"errors"
+	"math/big"
+)
+
+func bigFromHex(hex string) *big.Int {
```

Comment:
We have a `hexutil` package to parse and marshal hex numbers. Please use that instead of redeclaring methods (especially that ignore parse errors).

Reply:
[Related method](https://github.com/ethereum/go-ethereum/blob/3666da8a4b99e67c8c00684ae837e140b797823d/common/hexutil/hexutil.go#L138) says that it doesn't like numbers above 256 bits. Instead I changed it to FromHex under common package.